draft-ietf-mimi-content.md

```diff
@@ -1373,12 +1374,46 @@ cases, and should not be considered the result of a malicious sender.
   - where a contentType is unrecognized or unsupported.
   - where a language tag is unrecognized or unsupported.
 
+## Rendering and authorization of edits and deletes
+
+This content format allows clients to send new versions of previously sent
+messages, effectively replacing ("editing") or retracting ("deleting") other
+a referenced message. The rendering of these "edits" and "deletes" are important
+from a security perspective.
+
+For example, if Alice writes "Bob, could I borow a pen?", and Bob reacts with a
+thumbs up emoji, Alice might edit this message with no change in meaning
+(correcting the spelling of "borrow"), or a dramatic change in meaning ("Bob,
+could I borrow your Ferrari this month?"). The receiver SHOULD indicate clearly
+that a received message has been edited or retracted. The receiver might:
+
+- offer an option to view previous versions of a message,
+- show a summarized or thumbnail version of a message referenced in a reply,
+- indicate clearly that a reply was to a previous version of a message than the
+most recent one.
+- show a detailed view of reaction indicating that some reactions referred to a
+previous version of the message.
+
+In addition, some groups may have special policies or permissions allowing
+specific types of edits or deletes. For example, a moderator in one room might
+be allowed to edit the topic of a message, but not modify the rest of the
+content. An administrator might be allowed to delete messages which violate the
+policy of the group. Receiving clients SHOULD NOT allow parties other than the
```

Comment:
General question: should this (and the following) "SHOULD" be a "MUST"? It seems odd to me at this level that there would be an (application set) policy that has a valid reason to be ignored. Surely if there is a valid reason to not follow an existing authorization policy, the policy itself should be updated, vs the override occurring here.

Reply:
A messaging app where everyone is encouraged to edit each other's messages (like a collaborative work space) might not need an explicit policy, but in the MIMI case, you are probably right.